### ISSUE TITLE
Make issue template 

### ISSUE THREAD

**Issue by bhavyastar** [kube-green/kube-green#252]
-> Issue template provides ease for the folks who want to open an issue.
-> It differentiates the various types of issues such as `Code, Bug, Docs, Others`
-> It also follows community standards.

**Comment by davidebianchi** [comment]
Yes, this is definitely something to add to the repo, thanks for the suggestion!
